docs/indepth/installation.rst

```diff
@@ -347,10 +347,37 @@ You must also install the ZFS package repository.
 The following commands will install the two repositories and the ``clusterhq-flocker-node`` package.
 Paste them into a root console on the target node:
 
-.. task:: install_flocker
+.. task:: install_flocker fedora-20
    :prompt: [root@node]#
 
-Installing ``clusterhq-flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
+Installing on CentOS 7
```

Comment:
The various cloud provider-specific installation instructions all end with a suggestion to jump to the generic Fedora 20 installation instructions.  Maybe they can be changed to give a link to a generic installation instruction section that starts with a mini-index with links to sections for each OS (Fedora 20, CentOS 7 for now, Ubuntu eventually, etc).  Then folks who want non-Fedora 20 don't get confused when they reach the end of the AWS/rackspace/etc sections.